File: atcoder/369/A.py
def main():
    a, b = map(int, input().split())
    cnt = 0
    if a == b:
        print(1)
    elif (a+b) % 2 == 0:
        print(3)
    else:
        print(2)
if __name__ == "__main__":
    main()

File: atcoder/369/A.cpp
#include <iostream>
using namespace std;
int main()
{
    int a, b;
    cin >> a >> b;
    int cnt = 0;
    if ((a - b) % 2 == 0)
    {
        cnt += 1;
    }
    if (a != b)
    {
        cnt += 2;
    }
    if (a == b)
    {
        cnt = 1;
    }
    cout << cnt << endl;
    return 0;
}